doc/man3/X509_check_purpose.pod

```diff
@@ -0,0 +1,69 @@
+=pod
+
+=head1 NAME
+
+X509_check_purpose - Checks certificate I<x's> purpose
+
+=head1 SYNOPSIS
+
+ #include <openssl/x509v3.h>
+
+ int X509_check_purpose(X509 *x, int id, int ca)
+
+=head1 DESCRIPTION
+
+This function checks if B<certificate> I<x> was created with the B<purpose>
```

Comment:
I think there is no reason to bold "certificate" and "purpose" here. I think they have particular meanings in the man page standards (perhaps @richsalz can enlighten me on when we are now supposed to use bold tags)

Reply:
I've removed all bold entries since it's unclear as to when one should bold. I personally like the idea of having anything that is a type, macro, or function in the code base highlighted with bold. I've noticed sometimes I miss the italics, but I could just be losing my eyesight on occasion...